Packs/UrlScan/Integrations/UrlScan/UrlScan.py

```diff
@@ -356,13 +426,22 @@ def format_results(client, uuid):
         response_img = requests.request("GET", screen_path, verify=client.use_ssl)
         stored_img = fileResult('screenshot.png', response_img.content)
 
-    demisto.results({
-        'Type': entryTypes['note'],
-        'ContentsFormat': formats['markdown'],
-        'Contents': response,
-        'HumanReadable': tableToMarkdown('{} - Scan Results'.format(url_query), human_readable),
-        'EntryContext': ec
-    })
+    dbot_score = Common.DBotScore(indicator=dbot_score.get('Indicator'), indicator_type=dbot_score.get('Type'),
+                                  integration_name=dbot_score.get('Vendor'), score=dbot_score.get('Score'),
```

Comment:
use the global also in line: `dbot_score['Vendor'] = 'urlscan.io'` (I could not comment there)